### PR TITLE
[2.7] bpo-33071: remove outdated PyPI docs (GH-13087)

### PR DESCRIPTION
**Pull request by shihai1991**
Patch by Kojo Idrissa.
(cherry picked from commit 1b4abcf302ff2c8f4d4881294510d48ba5186b53)

Co-authored-by: Kojo Idrissa <kojoidrissa@users.noreply.github.com>

https://bugs.python.org/issue33071

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Our records indicate we have not received your CLA. For legal reasons we need you to sign this before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.

If you have recently signed the CLA, please wait at least one business day
before our records are updated.


You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for your contribution, we look forward to reviewing it!

File: Doc/distutils/packageindex.rst [hunk 1]
```diff
@@ -8,244 +8,10 @@
 The Python Package Index (PyPI)
 *******************************
 
-The `Python Package Index (PyPI)`_ stores :ref:`meta-data <meta-data>`
-describing distributions packaged with distutils, as well as package data like
-distribution files if a package author wishes.
-
-Distutils provides the :command:`register` and :command:`upload` commands for
-pushing meta-data and distribution files to PyPI, respectively.  See
-:ref:`package-commands` for information on these commands.
```
Comment (merwok) [inline Doc/distutils/packageindex.rst:17]: IMO most of the text should be kept, only without reference to register.

**Comment by csabella** [comment]
@shihai1991, since this is a manual backport to 2.7, please preface the PR title with `[2.7]`.  Thanks!

**Comment by csabella** [comment]
Also requesting review from @ncoghlan as he merged the pull request to master and 3.7.

**Comment by shihai1991** [comment]
> @shihai1991, since this is a manual backport to 2.7, please preface the PR title with `[2.7]`. Thanks!

copy that.

File: Doc/distutils/packageindex.rst (hunk 1 above)
Comment (shihai1991) [reply Doc/distutils/packageindex.rst:17]: done

**Comment by merwok** [comment]
I added my comment about removing too much info on the bpo issue, given that this is a backport PR so less appropriate for the discussion.

**Review by auvipy — APPROVED** [review]

**Comment by merwok** [comment]
@ncoghlan answered on the original PR: https://github.com/python/cpython/pull/13087#discussion_r288122908

Which means: please go back to the original version.  Sorry for the trouble!


A process note: rebases or force-pushes are not useful on CPython repo; there can be as many intermediary commits as needed, even with broken tests or docs, because we always do a squash merge in the end.  Non-rewritten commits also work a lot better in github’s UI, which will show «changes since last time you viewed» for reviewers.

**Comment by shihai1991** [comment]
copy that, let me try it again.

**Review by merwok — APPROVED** [review]

**Comment by merwok** [comment]
Thanks for the backport!

**Pull request merged**